### ISSUE TITLE
Not being able to add my self hosted email address.

### ISSUE THREAD

**Issue by smegtor** [LukeSmithxyz/mutt-wizard#484]
Yesterday I followed luke's guide on how to set up your own email server and everything went smoothly, 
I was able to login in thunderbird but when I tried it with mutt-wizard everytime it would fail and it includes an error `IMAP command 'LOGIN <user> <pass>' returned an error: NO [AUTHENTICATIONFAILED] Authentication failed.`
Also something weird that happens is when I enter my email address the error `gpg: error reading key: No public key` pops up but lets me continue with the setup.
I made sure I typed in the right IMAP and SMTP info and the correct password, 
it worked fine on thunderbird so its probably something with the missing public key which I don't know how to solve.

Thank you.

**Comment by thumbtackjake** [comment]
Same issue here.

**Comment by LukeSmithxyz** [comment]
Do you not have a GPG key pair?

**Comment by nexusrna** [comment]
I'm having this issue too. I checked in Thunderbird, and it was fine. That gpg related alert happened to me in the first time, when i still had not generated keys for the new address. Though, the imap error persists even after generating a keypair for the right email.

**Comment by XAWalter** [comment]
I dont know if this is a good fix or not. But a the very least I believe that the source of the error may stem from the imap and smtp server names. I switched my imap and smtp server from `[username]@[domain_name]@mail.[domain_name]` to `[username]@mail.[domain_name]`. Then I ensured to add the line `set from = "[username]@[domain_name]". Again I don't know if this is a good fix but I'll keep you guys updated if I find out anything else.

**Comment by XAWalter** [comment]
So I got it to work without having offline mail. Went into the script before running make install and deleted the `tryconnect` from the `add` case. After finalizing, I went into `account/[accountname]` in the muttrc configs and changed the `set folder` line and added `set smtp_url` with the change from my last post. I also added `set imap_pass = "\'pass mutt-wizard [username]\`"`.

There is a HUGE chance that I set up something wrong with my email server, domain name, or with mutt itself so take all of this with a grain of salt. I have very little experience with shell scripting and am way out of my knowledge zone. I hope that this is at least somewhat helpful.
